code/datums/components/crafting/recipes.dm

```diff
@@ -387,7 +387,7 @@
 
 
 /datum/crafting_recipe/radiogloves
-	name = "Translation Gloves"
+	name = "Sign Language Radio Gloves"
```

Comment:
I get this issue here but this might be a bit far. Thoughts?

Reply:
I wanted to make sure that people know this is for sign language, and not some kind of different translators, however I 100% agree I might have made it too long of a name. Perhaps I could simply call them "radio gloves" instead.

Reply:
mmm